.create-nuget.sh

```diff
@@ -0,0 +1,115 @@
+    msbuild /v:m /p:configuration=release /p:platform="anyCpu" /p:WarningLevel=0 Xamarin.Forms.Platform.Android/Xamarin.Forms.Platform.Android.csproj
+    msbuild /v:m /p:configuration=release /p:platform="anyCpu" /p:WarningLevel=0 Xamarin.Forms.Platform.Android/Xamarin.Forms.Platform.Android.csproj /p:AndroidTargetFrameworkVersion=v8.1 /t:Restore
+    msbuild /v:m /p:configuration=release /p:platform="anyCpu" /p:WarningLevel=0 Xamarin.Forms.Platform.Android/Xamarin.Forms.Platform.Android.csproj /p:AndroidTargetFrameworkVersion=v8.1
+    ;;
+  "pddroid")
+    CONFIG=Debug
+    msbuild /v:m /p:configuration=debug /p:platform="anyCpu" /p:WarningLevel=0 Xamarin.Forms.Platform.Android/Xamarin.Forms.Platform.Android.csproj
+    msbuild /v:m /p:configuration=debug /p:platform="anyCpu" /p:WarningLevel=0 Xamarin.Forms.Platform.Android/Xamarin.Forms.Platform.Android.csproj /p:AndroidTargetFrameworkVersion=v8.1 /t:Restore
+    msbuild /v:m /p:configuration=debug /p:platform="anyCpu" /p:WarningLevel=0 Xamarin.Forms.Platform.Android/Xamarin.Forms.Platform.Android.csproj /p:AndroidTargetFrameworkVersion=v8.1
+    ;;
+  "ios")
+    CONFIG=Debug
+    sh .create-stubs.sh $CONFIG
+    wine $NUGET_EXE restore .xamarin.forms.ios.nuget.sln
+    msbuild /v:m /p:platform="any cpu" .xamarin.forms.ios.nuget.sln
+    ;;
+  "droidios")
+    CONFIG=Debug
+    sh .create-stubs.sh $CONFIG
+    wine $NUGET_EXE restore .xamarin.forms.android.nuget.sln
+    wine $NUGET_EXE restore .xamarin.forms.ios.nuget.sln
+    msbuild /v:m /p:platform="any cpu" /p:WarningLevel=0 .xamarin.forms.android.nuget.sln
+    msbuild /v:m /p:platform="any cpu" .xamarin.forms.ios.nuget.sln
+    ;;
+  "all")
+    CONFIG=Debug
+    sh .create-stubs.sh $CONFIG
+    wine $NUGET_EXE restore .xamarin.forms.nuget.sln
+    msbuild /v:m /p:platform="any cpu" /p:WarningLevel=0 .xamarin.forms.nuget.sln
+    ;;
+  "rall")
+    CONFIG=Release
+    sh .create-stubs.sh $CONFIG
+    wine $NUGET_EXE restore .xamarin.forms.nuget.sln
+    msbuild /v:m /p:platform="any cpu" /p:WarningLevel=0 .xamarin.forms.nuget.sln /p:configuration=release
+    msbuild /v:m /p:platform="any cpu" /p:WarningLevel=0 .xamarin.forms.nuget.sln /p:configuration=release /p:AndroidTargetFrameworkVersion=v8.1 /t:Restore
+    msbuild /v:m /p:platform="any cpu" /p:WarningLevel=0 .xamarin.forms.nuget.sln /p:configuration=release /p:AndroidTargetFrameworkVersion=v8.1
+    ;;
+  *)
+    CONFIG=Debug
+    sh .create-stubs.sh $CONFIG
+    wine $NUGET_EXE restore .xamarin.forms.nuget.sln
+    msbuild /v:m /p:platform="any cpu" /p:WarningLevel=0 .xamarin.forms.nuget.sln
+    ;;
+esac
+
+DEBUG_VERSION=0
+wine $NUGET_EXE pack .nuspec/Xamarin.Forms.nuspec -properties configuration=$CONFIG\;platform=anycpu -Version 9.9.$DEBUG_VERSION
+if [ ! -n $CREATE_MAP_NUGET ]; then
+  # Requires building x86, x64, AMD
+  wine $NUGET_EXE pack .nuspec/Xamarin.Forms.Maps.nuspec -properties configuration=$CONFIG\;platform=anycpu -Version 9.9.$DEBUG_VERSION
```

Comment:
Does this need wine to work?  Nuget is installed as part of the mono framework and should be able to pack the nuspecs

Plus we should be able to just use 
*msbuild /t:restore" instead of *wine nuget_exe*

Reply:
You're right. I don't know what I was thinking about.
Question. Can I use msbuild for nuget packaging? I see in docs that it is possible. For example `msbuild -t:pack <path to .csproj file> -p:NuspecFile=<path to nuspec file> -p:NuspecProperties=<> -p:NuspecBasePath=<Base path>` but I cannot pick up right parameters.

Reply:
You might need to import this
> you must import NuGet.Build.Tasks.Pack.targets 

https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets#packing-using-a-nuspec

But might not work until those projects support sdk style or are converted to using SDK.Extras